tools/scylla-nodetool.cc

```diff
@@ -2841,6 +2874,23 @@ const std::map<std::string_view, std::string_view> option_substitutions{
 std::map<operation, operation_func> get_operations_with_func() {
 
     const static std::map<operation, operation_func> operations_with_func {
+        {
+            {
+                "backup",
+                "copy SSTables from a specified keyspace's snapshot to a designated bucket in object storage",
+fmt::format(R"(
+For more information, see: {}"
+)", doc_link("operating-scylla/nodetool-commands/backup.html")),
```

Comment:
Dangling link?

Reply:
should be fixed by https://github.com/xemul/scylla/pull/3